src/content/service.js

```diff
@@ -15,53 +16,94 @@ const getShortUid = () =>
     .join('')
     .substr(0, 6)
 
-module.exports = ({ db, botfile, projectLocation, logger }) => {
-  let categories = []
+const getNewItemId = category => {
+  const prefix = (category.ummBloc || category.id).replace(/^#/, '')
+  return `${prefix}-${getShortUid()}`
+}
 
-  const scanAndRegisterCategories = async () => {
-    categories = []
+const prepareDb = async () => {
+  const knex = getInMemoryDb()
+
+  await knex.schema.createTable('content_items', table => {
+    table.string('id').primary()
+    table.text('data')
+    table.text('formData')
+    table.text('metadata')
+    table.string('categoryId')
+    table.text('previewText')
+    table.string('createdBy')
```

Comment:
I think I was drunk when I named these initially; everywhere else we use `created_on` notation. Can we change it to stay consistent?

Reply:
I renamed them myself for two reasons:
1) the rest of the fields are camelCase
2) we want them camelCased in JSON and I wanted to save on transformation code

This is in-memory DB so why bothering?:)

Reply:
Ah I thought it was my original code, sorry! Since that's in-memory, let's keep it this way. Just keep in mind in all real tables we use the `a_b` notation :)